src/PrimPlane.h

```diff
@@ -43,9 +43,10 @@ class CPrimPlane : public IPrim
 
 	virtual CBoundingBox getBoundingBox(void) const override
 	{
-		CBoundingBox bounds;
-		// --- PUT YOUR CODE HERE ---
-		return bounds;
+	    // Planes are infinite so the bounding box is infinite?
+        Vec3f minPoint = Vec3f::all(-Infty);
+        Vec3f maxPoint = Vec3f::all(Infty);
+        return CBoundingBox(minPoint, maxPoint);
```

Comment:
This code is needed as well in case if the plane is orthogonal to one of the axises
```
		for (int i = 0; i < 3; i++)
			if (m_normal.val[i] == 1) {
				minPoint.val[i] = m_origin.val[i];
				maxPoint.val[i] = m_origin.val[i];
				break;
			}
```